test-data/message/verification-gossip-also-sent-to-from.eml

```diff
@@ -0,0 +1,123 @@
+Content-Type: multipart/encrypted; protocol="application/pgp-encrypted"; 
+	boundary="1854e0cc03e1678e_27ea5f4f17e5cce_dadff6845d0ea716"
+MIME-Version: 1.0
+From: <alice@example.org>
+To: <bob@example.net>, <fiona@example.net>, 
+	<alice@example.org>
+Subject: [...]
+Date: Thu, 1 Jan 1970 00:00:00 +0000
+Message-ID: <rfc724_mid>
```

Comment:
Does this really work? Maybe put some real random id here?

Reply:
Why wouldn't it work? From the view of the code, it's as good as any combination of letters.